packages/devtools_app/lib/src/framework/release_notes/release_notes-latest.json

```diff
@@ -0,0 +1,56 @@
+{
+     "version": {
+          "major": 2,
+          "minor": 18,
+          "patch": 0,
+          "pre": null
+     },
+     "sections": [
+          {
+               "name": "Inspector updates",
+               "notes": [
+                    {
+                         "githubPullRequestUrls": [
+                              "https://github.com/flutter/devtools/pull/4283"
+                         ],
+                         "message": "Auto scrolling behavior improved when snapping to a widget into focus"
```

Comment:
how can we add images to this file? can you add an example of that

Reply:
I was hoping to do that in a follow up, but it isn't hard to do now so I'll just add that now :D